test/ng/compileSpec.js

```diff
@@ -2762,6 +2762,16 @@ describe('$compile', function() {
     }));
 
 
+    it('should handle consecutive text elements as a single text element', inject(function($rootScope, $compile) {
+      var base = jqLite('<div></div>');
+      base[0].appendChild(document.createTextNode('Lucas {{1+'));
+      base[0].appendChild(document.createTextNode('2}} Lucas'));
```

Comment:
I think we should test the specific situation that causes the bug, i.e. that there is a mutation observer and that the text contains some special characters that trigger IE11 to split it into multiple text nodes.